examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala

```diff
@@ -203,7 +203,7 @@ object SparkSQLExample {
     // No pre-defined encoders for Dataset[Map[K,V]], define explicitly
     implicit val mapEncoder = org.apache.spark.sql.Encoders.kryo[Map[String, Any]]
     // Primitive types and case classes can be also defined as
-    implicit val stringIntMapEncoder: Encoder[Map[String, Int]] = ExpressionEncoder()
+    // implicit val stringIntMapEncoder: Encoder[Map[String, Any]] = ExpressionEncoder()
```

Comment:
This compiled before though, right? you're saying it's unnecessary because of the implicit in the line above? that seems fine but let's delete these 2 lines then.